src/templates/person/person.js

```
@@ -17,7 +17,7 @@ class PersonPage extends React.Component {
     this.title = `${props.firstName + ' ' + props.lastName ||
       ''} [${props.intro || ''}]`
     this.altTitle = `${props.firstName + ' ' + props.lastName ||
-      ''} [${props.altIntro || ''}]`
+      ''} [${props.altIntro || props.intro}]`
```

does this just fail silently if neither are entered?

---

good question - it wasn't supposed to be completely empty as it caused a glitch on hover. Probably needs further thought.